packages/pods-core/src/builtin/EmailPod.ts

```diff
@@ -15,49 +17,46 @@ export class EmailPublishPod extends PublishPod<EmailPublishConfig> {
   static id: string = ID;
   static description: string = "publish to email";
 
-  get config() {
-    return super.config.concat([
-      {
-        key: "from",
-        description: "from address",
-        type: "boolean",
-        default: false,
-        example: "you <username@outlook.com>",
+  get config(): JSONSchemaType<EmailPublishConfig> {
+    return PodUtils.createPublishConfig({
+      required: [],
+      properties: {
+        from: {
+          description: "from address",
+          type: "string",
+          default: false,
```

Comment:
Sorry one more small thing - is `false` a valid default value for a string? or does it need to be an empty string or something like that?

Reply:
`false` should not be a default value for string. My bad I didn't see it while changing the type from boolean to string.